### PR TITLE
Create data model

### PR DESCRIPTION
**Pull request by kiboma2021**
- Create a new app. Give it a meaningful name.
- Initialize your project with Git.
- Make sure that your project has Postgres database set up.
- Build your project schema.

- [ ] Create and run the necessary migration files.
- [ ] Table and column names should match the ERD diagram.
- [ ] Foreign keys should be included.
- [ ] All columns that are foreign keys should have a corresponding index.

image

### CODE REVIEW

File: db/migrate/20221017202611_add_user_ref_to_comments.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true, index: true
```
Comment (VICTOR-CHUKWU) [inline db/migrate/20221017202611_add_user_ref_to_comments.rb]: - [ ] According to the project requirement kindly change the reference to user to author i.e `  t.references :author, foreign_key: { to_table: 'users' }`

File: db/migrate/20221017202912_add_user_ref_to_likes.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true, index: true
```
Comment (VICTOR-CHUKWU) [inline db/migrate/20221017202912_add_user_ref_to_likes.rb]: - [ ] According to the project requirement kindly change the reference to user to author i.e `t.references :author, foreign_key: { to_table: 'users' }`

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @kiboma2021  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20221017202912_add_user_ref_to_likes.rb (hunk 2 above)
Comment (kiboma2021) [reply db/migrate/20221017202912_add_user_ref_to_likes.rb]: @VICTOR-CHUKWU , This is done

**Review by ShadyShawkat — APPROVED** [review]
## Hi @kiboma2021, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**